java/trino/src/main/java/sleeper/trino/SleeperSplitSource.java

```diff
@@ -105,10 +104,10 @@ private static List<SleeperSplit> generateSleeperSplits(SleeperConnectionAsTrino
     }
 
     /**
-     * Create a {@link CompletableFuture} where the {@link DynamicFilter} has narrowed as far as it possibly can.
+     * Create a future which will wait until filter has narrowed as far as it possibly can.
```

Comment:
The `the` has been dropped from this comment: `Creates a future which will wait until the filter has narrowed as far as it possibly can`

Reply:
Fixed, thanks